src/components/Home/blocks/Activities/Activities.tsx

```diff
@@ -0,0 +1,31 @@
+import { get, size } from "lodash";
+import { Title } from "@deskpro/app-sdk";
+import { NoFound } from "../../../common";
+import { isLast } from "../../../../utils";
```

Comment:
Instead of relative paths, would an imports alias be easier to maintain, eg `import { isLast } from "@/utils";`

Reply:
`NoneFound` is more accurate here as we're talking about a collection of things.